### ISSUE TITLE
releasing a new version based on the v1.4 branch

### ISSUE THREAD

**Issue by lboue** [home-assistant-libs/chip-wheels#103]
Hi,

Would you consider releasing a new version based on the v1.4 branch of the SDK?

This will help with python-matter-server: https://github.com/home-assistant-libs/python-matter-server/pull/950

**Comment by agners** [comment]
> [Release a new version for chip-build container](https://github.com/project-chip/connectedhomeip/issues/36468)

Not sure what you mean by that, but chip-build containers get uploaded whenever a new version comes out.

We do native builds for aarch64, so I had to build the aarch64 variant here. This is currently a manual process :cry: , but it's straight forward, using the same Dockerfile as the x86-64, so:

```
cd integrations/docker/images/base/chip-build
docker build --platform linux/arm64/v8 --tag agners/aarch64-chip-build:81 .
docker push agners/aarch64-chip-build:81
```

**Comment by lboue** [comment]
Thanks.
Fixed by #106.

**Comment by andy31415** [comment]
We generally do not release containers for branches.
